src/ui/MissingTerm/MissingTerms.ts

```diff
@@ -280,9 +282,13 @@ export class MissingTerms extends Component {
     });
   }
 
+  private get breakingCharacters() {
+    return `[-'?\*’.~=,\/\\\\:\`;_!&\(\)]+`;
```

Comment:
The index team will update us with their list of characters by the end of the day